### ISSUE TITLE
Error 500 in Services

### ISSUE THREAD

**Issue by lvicainne** [shinken-monitoring/mod-webui#607]
Hello !
I have  500 errors using the servcies page anf WebUi2 2.7 (I was using WebUi2 2.4 and it was working fine)

In "/var/lib/shinken/modules/webui2/helper.py", I commented the incriminated lines :
```
            #if p.uom != '%' and used_pct is not None:
            #    title += " ({:.2f}%)".format(used_pct)
```

Thus, it avoids me to get the following error.
Thanks for your code and your project !

> Error: 500 Internal Server Error
> 
> Sorry, the requested URL 'http://shinken:7767/all?search=type:service%20stag:service' caused an error:
> 
> Internal Server Error
> Exception:
> 
> ValueError('zero length field name in format',)
> Traceback:
> 
> Traceback (most recent call last):
>   File "/usr/lib/python2.6/site-packages/bottle.py", line 862, in _handle
>     return route.call(**args)
>   File "/usr/lib/python2.6/site-packages/bottle.py", line 1732, in wrapper
>     rv = callback(*a, **ka)
>   File "/var/lib/shinken/modules/webui2/module.py", line 509, in lock_version
>     return f(**args)
>   File "/usr/lib/python2.6/site-packages/bottle.py", line 3619, in wrapper
>     return template(tpl_name, **tplvars)
>   File "/usr/lib/python2.6/site-packages/bottle.py", line 3595, in template
>     return TEMPLATES[tplid].render(kwargs)
>   File "/usr/lib/python2.6/site-packages/bottle.py", line 3399, in render
>     self.execute(stdout, env)
>   File "/usr/lib/python2.6/site-packages/bottle.py", line 3386, in execute
>     eval(self.co, env)
>   File "/var/lib/shinken/modules/webui2/plugins/problems/views/problems.tpl", line 155, in <module>
>     {{!helper.get_perfdata_pies(pb)}}&nbsp;
>   File "/var/lib/shinken/modules/webui2/helper.py", line 449, in get_perfdata_pies
>     return " ".join([self.get_perfdata_pie(p) for p in PerfDatas(elt.perf_data)])
>   File "/var/lib/shinken/modules/webui2/helper.py", line 443, in get_perfdata_pie
>     title += " ({:.2f}%)".format(used_pct)
> ValueError: zero length field name in format

**Comment by maethor** [comment]
Thank you for reporting.

I'm not sure to understand what is causing this error, but I will look into it.

**Comment by lvicainne** [comment]
Hello @maethor  !
Thanks for your reply !
To give you more details, it is because the plugin returns some bad values after the "|" (instead of number, it is a string).
Anyway, I propose to modify these 2 lines by checking if used_pct is a float instead of check if it's not None :-)

**Comment by maethor** [comment]
I don't see why `used_pct` would be a string. I will fix it :)

**Comment by lvicainne** [comment]
Hi !
The source problem is from one of my plugins ;-) which return a string instead of a number.
But if WebUI2 can be resilient in case of bad plugins, it's better ! :-)

**Comment by maethor** [comment]
Oh, ok, thank you, I understand now.

**Comment by maethor** [comment]
I really don't understand how `used_pct` would be anything else than a float, but I changed the check. It will be fixed in 2.7.3
